package.json

```diff
@@ -15,10 +15,20 @@
   "main": "index",
   "repository": {
     "type": "git",
-    "url": "https://github.com/Yvem/node-exif.git"
+    "url": "https://github.com/jmunox/node-exif.git"
```

Comment:
It's completely fine publishing you own version on npm (open source rulz !) However, you should not include the name/repo change in the PR to the initial repo